cypress/fixtures/console/application/integrations/webhook/template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "templates": [{
+    "ids":{
+      "template_id":"akenza"
+    },
+    "name":"Akenza Core",
```

Comment:
Please use placeholder names, brands, etc. Check e.g. how we did it in `cypress/fixutres/devices/*.json`.

For the logo URL you can use a data URI: `data:image/gif;base64,R0lGODdhEAAQAMwAAPj7+FmhUYjNfGuxYYDJdYTIeanOpT+DOTuANXi/bGOrWj6CONzv2sPjv2CmV1unU4zPgISg6DJnJ3ImTh8Mtbs00aNP1CZSGy0YqLEn47RgXW8amasW7XWsmmvX2iuXiwAAAAAEAAQAAAFVyAgjmRpnihqGCkpDQPbGkNUOFk6DZqgHCNGg2T4QAQBoIiRSAwBE4VA4FACKgkB5NGReASFZEmxsQ0whPDi9BiACYQAInXhwOUtgCUQoORFCGt/g4QAIQA7`